command/root.go

```diff
@@ -219,7 +219,8 @@ func changelogURL(version string) string {
 func determineBaseRepo(cmd *cobra.Command, ctx context.Context) (ghrepo.Interface, error) {
 	repo, err := cmd.Flags().GetString("repo")
 	if err == nil && repo != "" {
-		return ghrepo.FromFullName(repo), nil
+		baseRepo, _ := ghrepo.FromFullName(repo)
```

Comment:
We should handle the error argument here, in case an invalid value was passed via the `--repo` flag

Reply:
done 👍